Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/TransformComponent.cpp

```diff
@@ -194,6 +194,7 @@ namespace AzToolsFramework
             , m_suppressTransformChangedEvent(false)
             , m_interpolatePosition(AZ::InterpolationMode::NoInterpolation)
             , m_interpolateRotation(AZ::InterpolationMode::NoInterpolation)
+            , m_focusModeInterface(AZ::Interface<AzToolsFramework::FocusModeInterface>::Get())
```

Comment:
You should check if this interface was actually initialized correctly (if there's no handler, Get will return nullptr)

Reply:
Added check.